packages/manager/fragments/account-sidebar/src/components/AccountSidebar.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <sidebar :closed="!sidebarOpen" class="manager-hub-user-panel">
+    <account-sidebar-user-infos class="mb-3" :user="user"></account-sidebar-user-infos>
+    <Suspense>
+      <template #default>
+        <account-sidebar-payment class="mb-4"></account-sidebar-payment>
+      </template>
+      <template #fallback>
+        <account-sidebar-payment-skeleton class="mb-4"></account-sidebar-payment-skeleton>
+      </template>
+    </Suspense>
+    <account-sidebar-shortcuts class="mb-4" :shortcuts="shortcutList"></account-sidebar-shortcuts>
+    <account-sidebar-links class="mb-5" :links="usefulLinks"></account-sidebar-links>
+  </sidebar>
+</template>
+
+<script lang="ts">
+import { defineAsyncComponent, defineComponent, inject, onMounted, ref, Ref } from 'vue';
+import { Environment } from '@ovh-ux/manager-config';
```

Comment:
`Environment` is no more a singleton on `develop`. With React we use Context (https://reactjs.org/docs/context.html) to make `environment` available everywhere.